arangosh/Export/ExportFeature.cpp

```diff
@@ -840,4 +839,23 @@ void ExportFeature::xgmmlWriteOneAtt(ManagedDirectory::File & fd,
   }
 }
 
+void ExportFeature::appendCsvStringValue(std::string& output, std::string const& value) {
+  // escape value and put it in quotes
+  output.push_back('\"');
+  // if we are going to emit a string, we have to take some security precautions.
+  // for example, to prevent formula injection in MS Excel and LibreOffice calc, any
+  // string cells starting with one of the characters =, +, -, @ need to be escaped
+  // with an extra single quote (') so that their contents will not be interpreted
+  // as formulae 🙄
+  // https://infosecwriteups.com/formula-injection-exploiting-csv-functionality-cd3d8efd02ec
+  if (_escapeCsvFormulae && !value.empty()) {
+    bool escapeFormula = value.front() == '=' || value.front() == '+' || value.front() == '-' || value.front() == '@';
+    if (escapeFormula) {
```

Comment:
it's just a bool so wouldn't make a difference in performance, but perhaps this?
``` 
static const unordered_set<char> escape_chars = {'=', '+', '-', '@'};
if (escape_chars.contains(value.front()) {
  output.push_back('\"');
}
```
to be easier to add more characters if needed